libr/reg/reg.c

```diff
@@ -202,6 +202,40 @@ static int regcmp(RRegItem *a, RRegItem *b) {
 	return offa > offb;
 }
 
+R_API void r_reg_rerole(RReg *reg) {
```

Comment:
This looks like an expensive operation. Shall it be always done? Also, please consider if existing API like `r_reg_get` or others can be used instead of hand-made for-loops.

Reply:
oh wait, ill wait for this to be fixed, my plan is to release later today or tomorrow. ill be fixing more things during the flight

Reply:
i killed it, found another way